Configurable guacd address

rokett opened the pull request:
This set of changes do 4 things.

1. Allows for configuring the guacd address and port as an input flag.  Fixes #13 
2. Replaces a deprecated function call from the stdlib
3. Small correction to the log entry for starting the web server; 0.0.0.0 instead of 127.0.0.1.
4. Hides passwords in log entries; previously they were printed in plaintext which may be picked up by logging platforms and stored.

terickson commented:
I like this option but instead of command line flags I think we should use [environment variables](https://gobyexample.com/environment-variables).  That way its much easier to inject them into docker containers, K8s, or however your hosting this application.  Also since we are adding options we should list how to use them in the readme file.

rokett commented:
That makes sense.  I've switched it to use an environment variable instead, and added some light info into the readme.

It likely shouldn't be a global variable, but it required the fewest number of changes to the existing code this way.  Let me know what you think?  For my use case, being able to configure the guacd address is important and I'd rather contribute back to the source than fork a project, so am happy to make further changes.

terickson commented:
This looks good.   I'm actually fine with the parameter and its default value being set at a global level.  I think it makes it more obvious to developers who are new to the code base, what the parameters are and where to define them, so in this case for readability, I'm fine with it.  Thanks for the contribution.

rokett commented:
that's great..thanks.  I'll amend my other PR accordingly.
